### PR TITLE
Xrddev 1807  CS address edit e2e test cases

### PR DESCRIPTION
**Pull request by teemukin65**

### CODE REVIEW

File: src/centralserver/admin-ui/tests/e2e/specs/040-cs-system_settings.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  'SystemSettings/System Parameters view has correct titles and ServerAddress edit opens correct dialog':
+    async (browser) => {
+      await settings
+        .systemSettingsViewIsVisible()
+        .verify.elementPresent('@systemSettingsParametersCard')
+        .verify.elementPresent('@systemSettingsParametersTitle')
+        .verify.elementPresent('@systemSettingsInstanceIdentifierFieldTitle')
+        .verify.containsText(
+          '@systemSettingsInstanceIdentifierFieldTitle',
+          'Instance Identifier',
```
Comment (Riippi) [inline src/centralserver/admin-ui/tests/e2e/specs/040-cs-system_settings.js:55]: Seems that there are a couple of tests for titles that are just localised text. Is there any benefit on testing those? 
I think we should check only the essential "moving parts". Also maybe all the data-test tags are not needed if they are just for checking texts.

**Review by Riippi — CHANGES_REQUESTED** [review]

File: src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue [hunk 2]
```diff
@@ -182,7 +199,7 @@
                   </div>
                 </div>
               </td>
-              <td>
+              <td data-test="">
```
[the remark this answers is not in the document]
Comment (teemukin65) [reply src/centralserver/admin-ui/src/views/Settings/SystemSettings/SystemSettings.vue]: fixed: 1589a4d , also removed all data-test entries for fixed titles. Added similarly to other fields in the component

File: src/centralserver/admin-ui/tests/e2e/specs/040-cs-system_settings.js (hunk 1 above)
Comment (teemukin65) [reply src/centralserver/admin-ui/tests/e2e/specs/040-cs-system_settings.js]: Tests for fixed texts removed. acc4a39

Comment (teemukin65) [reply src/centralserver/admin-ui/tests/e2e/specs/040-cs-system_settings.js]: And test case passes still locally.

**Review by Riippi — APPROVED** [review]

**Pull request merged**